docs/admin/cluster-management.md

```diff
@@ -33,7 +33,9 @@ Documentation for other releases can be found at
 
 # Cluster Management
 
-This doc is in progress.
+## Creating and configuring a Cluster
+
+Assuming that you have machines for your clusters ready, you need to install Kubernetes on them. For specific instructions you can consult one of existing [Getting Started guides](../../docs/getting-started-guides/README.md) for various environments.
```

Comment:
Please change to just "To install Kubernetes on a set of machines, consult one of the existing [Getting Started guides](../../docs/getting-started-guides/README.md) depending on your environment."